src/main/java/seedu/address/logic/commands/DeleteCommand.java

```diff
@@ -8,7 +8,7 @@
 import seedu.address.commons.core.index.Index;
 import seedu.address.logic.commands.exceptions.CommandException;
 import seedu.address.model.Model;
-import seedu.address.model.person.Person;
+import seedu.address.model.internship.Internship;
 
 /**
  * Deletes a person identified using it's displayed index from the address book.
```

Comment:
Change the name of the class to `DeleteCommand1` , so we do not mess AddressBook's implementations. 
Thanks